xc7/tests/dram/CMakeLists.txt

```diff
@@ -21,15 +27,15 @@ function(dram_tests dram_num_instances dram_mode outpad inpad)
 
 endfunction()
 
-# dram_tests(<dram_num_instances> <dram_mode> <syn_outpad_assert_usage> <syn_inpad_assert_usage>)
-dram_tests(4 32x1s 4 11)
-dram_tests(8 32x1s 8 11)
-dram_tests(2 32x1d 8 15)
-dram_tests(4 32x2s 4 14)
-dram_tests(2 64x1d 4 12)
-dram_tests(4 64x1s 4 16)
-dram_tests(2 128x1s 2 11)
-dram_tests(1 128x1d 2 17)
-dram_tests(1 256x1s 1 11)
-dram_tests(1 32m 8 17)
-dram_tests(1 64m 4 17)
+# dram_tests(<dram_num_instances> <dram_mode> <syn_outpad_assert_usage> <syn_inpad_assert_usage> <enable_vivado_target>)
+dram_tests(4 32x1s 4 11 N)
```

Comment:
Why are the `32x1s`/`32x1d` failing?  Please fix these two.

Reply:
Regarding 32x1d it is currently enabled and completes correctly (`dram_2_32x1d_vivado_diff_fasm`).
As for 32x1s, vivado fails with the following error:
```
ERROR: [Vivado 12-2285] Cannot set LOC property of instance 'CLBLM_R_X3Y116_SLICE_X2Y116_RAM32X1S_C_0',  for bel C5LUT Two RAM symbols can not share a LUT site if two RAM symbols are not also sharing the D LUT site. There are fewer than two RAM symbols in the D LUT site,  for bel D5LUT Both D6LUT and D5LUT are occupied by a RAM. This prevents LUT xxx to go to site yyy
Resolution: When using BEL constraints, ensure the BEL constraints are defined before the LOC constraints to avoid conflicts at a given site.

```

I have tried to adopt the `Resolution` suggested, but the same failure is produced.